Px.Utils.TestingApp/Commands/PxFileValidationBenchmark.cs

```diff
@@ -26,7 +26,15 @@ protected override void OneTimeBenchmarkSetup()
 
             using Stream stream = new FileStream(TestFilePath, FileMode.Open, FileAccess.Read);
             PxFileMetadataReader reader = new();
-            encoding = reader.GetEncoding(stream);
+            try
+            {
+                encoding = reader.GetEncoding(stream);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine($"Error while reading the encoding of the file: {e.Message}");
```

Comment:
I think It would be better if the GetEncoding method throws just re-throw here instead of wrapping the message.